### ISSUE TITLE
Hello

### ISSUE THREAD

**Issue by gngchrs** [nairobi-gophers/fupisha#1]
Hi, since at least 12 people are watching the repo. Can we push to master via pull requests. It's hard to follow what is being built.

**Comment by basebandit** [comment]
Hello chris. Sure yes..I will share the project plan as soon as I get time today.

**Comment by basebandit** [comment]
Hey I am working on a draft document with regards to project layout and the restful API .So far here is the progress. It is my hope that more people will get involved in the process. Currently I have put on hold any coding to first straighten out this important phase. Here are links to the [project layout](docs/project_layout.md) and [project api](docs/api.md) respectively. Any suggestions ideas, and more so pull requests are highly appreciated.

**Comment by gngchrs** [comment]
@basebandit thanks for the prompt feedback. I'll close this for now
